.rubocop.yml

```diff
@@ -4,7 +4,6 @@ inherit_from:
 
 AllCops:
   DisplayCopNames: true
-  TargetRubyVersion: 2.2
   Exclude:
```

Comment:
So my understanding is that Rubocop and Circle should both read from the `.ruby-version` file, now that we're including it.

http://rubocop.readthedocs.io/en/latest/configuration/#setting-the-target-ruby-version